.github/workflows/frontend_tests.yml

```diff
@@ -28,3 +28,9 @@ jobs:
         run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
       - name: Run frontend tests
         run: PYTHONIOENCODING=utf-8 python -m scripts.run_frontend_tests --run_minified_tests --skip_install --check_coverage
+      - name: Report failure if failed on oppia/oppia develop branch
+        if: ${{ github.event_name == 'push' && github.repository == 'oppia/oppia' && github.ref == 'refs/heads/develop'}}
```

Comment:
Is `github.ref` ever `refs/heads/develop` when `github.event_name` is not `push`?

Reply:
This workflow only runs on `push` & `pull_request`, the `ref` for pull_request have the following pattern `/ref/oppia/<pr-id-(number)>` So I confirm `refs/heads/develop` will be possible only on `push` events!